zdocs/quickstart.cheatmd

```diff
@@ -446,3 +446,14 @@ We can even get the overlapping area as its own range!
 iex> Range.intersection!(a, b)
 "<01:45:00:00 - 02:00:00:00 :exclusive <23.98 NTSC>>"
 ```
+
+## Postgres Types
+
+To use Postgres types with Ecto. Add the following into you applications configuration
```

Comment:
```suggestion
To include Postgres types with Ecto. Add the following into you applications configuration
```

Reply:
In 3374abf932f847793ed6277ab20c3ab0df2cef14